x-pack/plugins/lens/public/xy_visualization/xy_config_panel/axis_settings_popover.tsx

```diff
@@ -314,30 +320,42 @@ export const AxisSettingsPopover: React.FunctionComponent<AxisSettingsPopoverPro
         checked={areTickLabelsVisible}
       />
       <EuiSpacer size="s" />
-      <EuiFormRow
-        display="rowCompressed"
-        fullWidth
-        label={i18n.translate('xpack.lens.xyChart.axisOrientation.label', {
-          defaultMessage: 'Orientation',
+      <TooltipWrapper
+        tooltipContent={i18n.translate('xpack.lens.xyChart.axisOrientationMultilayer.disabled', {
+          defaultMessage:
+            'This option is only available for categorical axes or the legacy time axis (see Advanced Settings -> visualization:useLegacyTimeAxis)',
```

Comment:
From here https://github.com/elastic/kibana/issues/117476#issuecomment-965161274 I see that @ghudgins proposes to not mention the legacy time axis.

Reply:
updated the copy according to Graham suggestions: https://github.com/elastic/kibana/pull/117828/commits/9b9540ac00ba4163436a534dc008bd5e60d6bc6b